docs/schemas/Library-Minimal-Metadata/Library Minimal Metadata.csv

```diff
@@ -0,0 +1,52 @@
+Proposed BICAN Field Name,BICAN UUID,Label,Variable (short) name,definition,short description,human readable label,is_required,Permissible Values,data type,version,version date,sub domain name,domain name,size,min value,max value,measurement type
+tissue sample label,2e4ca2fc-2d77-4d19-af45-d0fb7bbc2269,tissue name,,Identifier name for final intact piece of tissue before cell or nuclei prep.  This piece of tissue will be used in dissociation and has an ROI associated with it.,This is the same field as tissue sample label in the donor to alignment metadata standard.,tissue sample label,TRUE,,text,1,9/9/23,Receive Sample,Library Minimal Metadata,,,,
+amplified cDNA label,e2606a11-114e-472f-9e05-33f9b6fc3089,amplified cdna name,,"Name of a collection of cDNA molecules derived and amplified from an input barcoded_cell_sample.  These cDNA molecules represent the gene expression of each cell, with all cDNA molecules from a given cell retaining that cell's unique barcode from the cell barcoding step.  This is a necessary step for GEX methods but is not used for ATAC methods.
+",,,TRUE,,Text,1,9/9/23,Generate Library,Library Minimal Metadata,,,,
+amplified cDNA amplified quantity ng,0db79d05-8612-4896-b9d3-eb1558841449,amplified quantity ng,,"Amount of cDNA produced after cDNA amplification measured in nanograms
+",,,TRUE,,Integer,1,9/9/23,Generate Library,Library Minimal Metadata,,,,

```

Comment:
amplified cDNA amplified quantity ng: expected data type -> float
data example provided in green table: 129.8939955

```suggestion
",,,TRUE,,Float,1,9/9/23,Generate Library,Library Minimal Metadata,,,,

```